### DISCUSSION TITLE
Possible to communicate between web workers with sockets locally?

### DISCUSSION THREAD

**Discussion by djytw** [emscripten-core/emscripten#20727] [Q&A] [answered]
If only local socket connections between modules/workers are needed, it seems weird to use a remote websocket proxy. Is there any existing way to implement this "loopback" posix socket between webworkers, maybe some way to proxy socket apis to worker apis?

**Comment by djytw** [comment]
Maybe not related, but similar issue #1422 (in my case it's INET socket)

**Comment by sbc100** [comment, marked as answer]
No we don't currently have support for anything like that.  

In fact, there is nothing in the form of IPC for different emscripten-built modules.   Currently would would have to do any wiring of multiple modules/instance yourself,  emscripten doesn't do any of that for you.

**Reply by djytw** [reply]
OK Thanks!
